ecs-cli/modules/utils/utils.go

```
@@ -139,12 +142,14 @@ func ParseLoadBalancers(flagValues []string) ([]*ecs.LoadBalancer, error) {
 				ContainerName:  aws.String(m["containerName"]),
 				ContainerPort:  aws.Int64((containerPort)),
 			})
-		} else {
+		} else if elbv1 {
```

also need to throw an error when both elbv1 and elbv2 are true.